Description: basic GNU/Hurd support
Last-Update: 2010-08-30
Index: lightspark/CMakeLists.txt
===================================================================
--- lightspark.orig/CMakeLists.txt	2010-08-30 14:32:20.000000000 +0200
+++ lightspark/CMakeLists.txt	2010-08-30 14:32:48.000000000 +0200
@@ -85,6 +85,13 @@
 	SET(ppc 1)
 	SET(LIB_SUFFIX "" CACHE STRING "Choose the suffix of the lib folder (if any) : None ppc")
 	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -maltivec")
+ELSEIF(${CMAKE_SYSTEM_PROCESSOR} MATCHES "unknown" AND ${CMAKE_SYSTEM} MATCHES "GNU-0.3")
+        SET(i386 1)
+        SET(LIB_SUFFIX "" CACHE STRING "Choose the suffix of the lib folder (if any) : None 32")
+        # nasm for assembly optimizations
+	IF (CMAKE_COMPILER_IS_GNUCC)
+	   ENABLE_LANGUAGE(ASM-NASM)
+	ENDIF ()
 ELSE()
 	MESSAGE(SEND_ERROR "Platform ${CMAKE_SYSTEM_PROCESSOR} not supported")
 ENDIF()
